cloud-infrastructure/storage/block-storage/asset/secure.storage.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+#
+# Version: @(#).secure.storage.sh 1.0.0 
+# License
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License (UPL), Version 1.0.
+# See [LICENSE](https://github.com/oracle-devrel/technology-engineering/blob/folder-structure/LICENSE) for more details.
+#
+#@  secure.storage.sh
+#@
+#
+# Update history:
+#
+# V 1.0.0 28.06.2023 initial version
+#
+
+# ---------------------------------------------------------------------------------------------------------------------------------------------
+# prepare environement (load functions)
+# ---------------------------------------------------------------------------------------------------------------------------------------------
+
+source functions0.sh
```

Comment:
Where to find this file?

Reply:
just uploaded both needed files parameter1.sh and functions0.sh